charts/shoot-core/components/charts/network-policies/templates/allow-from-to-nginx.yaml

```diff
@@ -0,0 +1,21 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  annotations:
+    gardener.cloud/description: |
+      Allows Egress and Ingress from the nginx-controller to all traffice
```

Comment:
```suggestion
      Allows all Egress and Ingress for the nginx-controller
```

Reply:
thanks @danielfoehrKn ! i've corrected as you commented.
-Neo